controller/handler/trial.js

```diff
@@ -124,6 +127,10 @@ function trialView (request, reply) {
 
             const endDate = processRules(ruleValues, Date.now());
 
+            if (request.query.requestType === 'ajaxRequest') {
```

Comment:
I am very uncomfortable about a single route being both a view and a resource API.
The chart API should really be its own route.